### PR TITLE
Add german translations

### PR DESCRIPTION
**Pull request by ComeIn-NRW**
PR to add german translations (and minor fixes, e.g. typos, in en-US)

### CODE REVIEW

File: lrmi_vocabs/alignmentType/alignmentType.ttl [hunk 1]
```diff
@@ -9,53 +9,53 @@
...
-    dct:modified "2017-12-20"^^xsd:date ;    
+    dct:modified "2017-12-20"^^xsd:date ;
     dct:license <http://creativecommons.org/licenses/by/4.0/> .
 
 alignment:assesses a skos:Concept ;
-    skos:prefLabel "assesses"@en-US ;
-    skos:definition "The learning resource being described may be used to assess the competency being referenced."@en-US ;
-    dct:source "Based on Common Education Data Standards (CEDS): https://ceds.ed.gov/element/000869#Assesses."@en-US ;
+    skos:prefLabel "assesses"@en-US, "testet"@de ;
+    skos:definition "The learning resource being described may be used to assess the competency being referenced."@en-US, "Die beschriebene Lernressource kann verwendet werden, um die adressierte Kompetenz zu testen."@de ;
```
Comment (acka47) [inline lrmi_vocabs/alignmentType/alignmentType.ttl:21]:
```suggestion
    skos:definition "The learning resource being described may be used to assess the competency being referenced."@en-US, "Die beschriebene Lernressource kann verwendet werden, um die referenzierte Kompetenz zu testen."@de ;
```

**Review by acka47 — APPROVED** [review]
I suggested one improvement, otherwise this makes a very good impression with regard to how the rather dry and difficult to read English definitions are translated to German. (I must admit, that I didn't have the time to go through it very thoroughly.) Anyway, the sentences are often also hard to read in German and we might in the future think about a way to provide easier to understand versions.

File: lrmi_vocabs/alignmentType/alignmentType.ttl (hunk 1 above)
Comment (philbarker) [reply lrmi_vocabs/alignmentType/alignmentType.ttl:21]: Thank you @acka47

**Comment by acka47** [comment]
What about merging this, @philbarker ?

**Comment by philbarker** [comment]
Thanks @acka47 No problem merging this and publishing on DCMI site. Sorry about the delay which came about when I found I needed to re-organise the DCMI pages in order to publish the Learning Resource Types -- I don't like to have too many moving parts at any one time. German version is next on my to do list.

**Pull request merged**